CHANGELOG.md

```diff
@@ -1,6 +1,46 @@
 ## master
 
-* [CHANGE] `node_cpu` metrics moved from `stats` to `cpu` collector on linux (enabled by default).
+## v0.15.0 / unreleased
```

Comment:
Confident enough to release 0.15.0 without a rc? Fine with both.

Reply:
I was unsure if we should cut a pre-release or not.  I know most of the features are already deployed "somewhere" from master, but maybe a month old.

Reply:
Do any point releases (e.g. 0.15.1) ever take place?  Appears that release candidates are more common, though I don't know the criteria that determines which should be used.

Reply:
We've done a bunch of RCs in the past, but it took a long time to get feedback on them.  This is why I'm thinking about just going right to release and posting patch releases if we do find any bugs.

Reply:
That makes sense to me. Since we aren't at 1.0 and don't have a formal release process (that I am aware of), I'm in favor of releasing now.

Reply:
SGTM